### DISCUSSION TITLE
Grammar correction

### DISCUSSION THREAD

**Discussion by GrampaWildWilly** [aclap-dev/video-downloadhelper#264] [License & Purchase requests]
### What's your issue?

There is bad grammar in the new discussion template.  You could say, "I confirm that I didn't paste . . ." or you could say, "I confirm that I did not paste . . ." which I think would be better.  But "didn't not" is simply not English.

### The 10 first digits (only 10) of your license number or email address

0123456789

### I confirm that I didn't not paste my full license number

- [X] I confirm

**Comment by Mattmjs** [comment]
I also think users posting email addresses in plain text on a public forum is a bad idea. There are spam bots programmed to
harvest email addresses across the internet. Then your address is suddenly on a spam mailing list.
The original poster should edit their post once their account problem is resolved & remove email, if not Jerome should do this.

**Reply by GrampaWildWilly** [reply]
Good point.  Although, Jérôme quite often needs the E-mail address to perform certain tasks related to the license.  Maybe there's a way to make posts in this section of the repository private, hidden.  This area is not of much interest to anybody but Jérôme & whoever might be asking for help.  I think an argument could be made that it doesn't have to be public.  Only people posting in this section should be able to see anything here, & that should be restricted to the discussions they've created themselves.

I've never been comfortable with showing even a part of the license number, either.  The one I posted above to satisfy the template is obviously fake.  I have always removed that part of the copy/paste when I've used it.  I've noticed a small number of our fellow users also removing it.  I voiced my opinion that I thought it was a bad idea the first time I saw it.  Maybe we can get this partial license removed not only from the new discussion template but also from the information copied to clipboard from the first page of settings.

**Reply by paulrouget** [reply]
We actually do not recommend people to post here. We pinned a post explaining that people should contact us directly: https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help#license-problem

And I agree, license number & emails should not be posted.

**Reply by paulrouget** [reply]
I'm going to add a "contact us directly" in the template.

Edit: done.

**Reply by GrampaWildWilly** [reply]
It's an improvement but I might append a more forceful statement to what you already have there.  Something along the lines of . . .

```
We encourage you to send us an E-mail at that address.  Most license issues can & should be handled by an exchange of E-mails between you & us.  It is probable that we will be able to solve your problem without you posting a new discussion here.  So assume it is preferable to E-mail us rather than posting something here.
```

I would remove the text entry box that prompts for the license number or E-mail address.  Don't even present a semblance of interest in those items.  But I would keep the check box.  It is important to make people explicitly acknowledge that posting a license number is a really bad idea.

**Comment by paulrouget** [comment]
I fixed the grammer issue.

**Reply by GrampaWildWilly** [reply]
Are you more or less grammer than I am?
